src/test/java/mops/domain/models/TimespanTest.java

```diff
@@ -0,0 +1,77 @@
+package mops.domain.models;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TimespanTest { //NOPMD
+    private Validation testValidator; //NOPMD
+
+    /**
+     * Erstellt leeren validator der in jedem Test angepasst wird.
+     */
+    @BeforeEach
+    public void setValidator() {
+        this.testValidator = Validation.noErrors();
+    }
+
+    @Test
+    @SuppressWarnings({"PMD.LawOfDemeter"}) //NOPMD
+    public void nullStartTime() {
+        final Timespan timespan = new Timespan(null, LocalDateTime.now());
+        this.testValidator = this.testValidator.appendValidation(new Validation(FieldErrorNames.TIMESPAN_START_NULL));
+        final Validation validator = timespan.validate();
+        assertThat(this.testValidator.getErrorMessages()).isEqualTo(validator.getErrorMessages());
```

Comment:
Ginge theoretisch auch einfacher:
`assertThat(this.testValidator.getErrorMessages()).containsOnly(FieldErrorNames.TIMESPAN_START_NULL);`
(oder so ähnlich)

https://joel-costigliola.github.io/assertj/core/api/org/assertj/core/api/ListAssert.html